### DISCUSSION TITLE
Support for hints

### DISCUSSION THREAD

**Discussion by gregy4** [ERGO-Code/HiGHS#867] [Q&A] [answered]
Hello,
  can mip solver in HiGHS benefit from feasible solution known before calculation ? In other solvers it is called hints or warm starts.
  In positive case, is it possible to use it now ?

Thanks,
Jan

**Comment by jajhall** [comment, marked as answer]
Yes, just use `Highs::setSolution` before you call `Highs::run` (or equivalent method in C API)

**Reply by jajhall** [reply]
Sorry I missed your question. Discussions are new to us

**Reply by gregy4** [reply]
Thanks. I hope it will be available also in interface to ortools.

**Reply by jajhall** [reply]
I hope so, once the OR-Tools interface has been written
